codegen/src/main/java/software/amazon/awssdk/codegen/model/intermediate/Metadata.java

```diff
@@ -499,6 +504,17 @@ public Metadata withJsonVersion(String jsonVersion) {
         return this;
     }
 
+    public Map<String, String> getAwsQueryCompatible() { return awsQueryCompatible; }
+
+    public void setAwsQueryCompatible(Map<String, String> awsQueryCompatible) {
+        this.awsQueryCompatible = awsQueryCompatible;
+    }
+
+    public Metadata withAwsQueryCompatible(Map<String, String> awsQueryCompatible) {
```

Comment:
Was looking for some confirmation that awsQueryCompatible is of key-value type String-String, the specification doesnot define the type for Map

Reply:
since it is an empty map anyway, string string map should suffice.

Reply:
It would be helpful in feature if we mention this in spec document.